rust_src/admin/remacs-helpers.el

```diff
@@ -0,0 +1,34 @@
+;;; Package -- helpers for Remacs porting
+
+(defun remacs-helpers/make-rust-args-from-C-worker (input)
+  "Transform function C arguments INPUT into Rust style arguments."
+  (mapconcat (lambda (arg) (let* ((is-struct (lambda (s) (string= s "struct")))
+                                  (pieces (cl-remove-if is-struct (split-string (string-trim arg) " ")))
```

Comment:
I used `cl-remove-if` because it allowed the list of ellided words to grow. I could pull `is-struct` out, rename it to "ignored-type-modifiers" and use that.